admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/constant/ZoneErrorCode.java

```diff
@@ -11,7 +11,7 @@ public enum ZoneErrorCode {
 	ZONE_FETCH_EXCEPTION("KER-MSD-337", "Error occured while fetching zone"),
 	USER_ZONE_UNAVAILABLE("KER-MSD-339", "No zone found for the logged-in user %s"),
 	USER_ZONE_FETCH_EXCEPTION("KER-MSD-338", "Error Occured while fetching zone of the user"),
-	ZONEUSER_ENTITY_NOT_FOUND("KER-MSD-391", "Zone mapping missing. Please contact the administrator."),
+	ZONEUSER_ENTITY_NOT_FOUND("KER-MSD-391", "Zone mapping missing. Kindly map the user to zone"),
```

Comment:
Did we change the message also in artifactory i18n?

Reply:
Yes.Please check this :https://github.com/mosip/artifactory-ref-impl/pull/455